bin/runner.js

```diff
@@ -37,9 +37,9 @@ var argv = yargs
       return require('./commands/init')(argv);
     }
   })
-  .command('build', Constants.cliMessages.BUILD.INFO, function(yargs) {
+  .command('build-info', Constants.cliMessages.BUILD.INFO, function(yargs) {
```

Comment:
are we changing this command name? We will have to update the documentation as well for this

Reply:
done